tests/system/libraries/WindowsLib.py

```diff
@@ -130,17 +139,23 @@ def taskSwitchToItemMatching(targetWindowNamePattern: _re.Pattern, maxWindowsToT
 			)
 
 
-def _tryOpenTaskSwitcher(pattern, spy):
-	nextIndex = spy.get_next_speech_index()
-	builtIn.log(f"Looking for window: {pattern}", level="DEBUG")
+def _tryOpenTaskSwitcher(spy: "_NVDASpyLib") -> _Optional["_SpeechIndexT"]:
+	"""
+	@param spy: The NVDA spy lib to be used.
+	@return: If the task switcher 'row 1' was spoken, the speech index for the start of the task switcher
+	speech.
+	"""

```

Comment:
Is there a reason the usual convention of `spy = _NvdaLib.getSpyLib()` is avoided here?

```suggestion
def _tryOpenTaskSwitcher() -> _Optional["_SpeechIndexT"]:
	"""
	@return: If the task switcher 'row 1' was spoken, the speech index for the start of the task switcher
	speech.
	"""
	spy = _NvdaLib.getSpyLib()
```

Reply:
Not really. I think the main reason for the convention was the uncertainty about how to provide type hints for the spy as a parameter. I'm happy to keep the convention.